src/sdks/core/src/cpp/sdk/cpptest/CoreSDKTest.cpp

```diff
@@ -727,7 +728,41 @@ EnumMap<Firebolt::Lifecycle::LifecycleEventSource> lifecycleEventSourceMap = {
     { Firebolt::Lifecycle::LifecycleEventSource::REMOTE, "remote" }
 };
 
-void CoreSDKTest::OnBackgroundNotification::onBackground(const Firebolt::Lifecycle::LifecycleEvent& lifecycleEvent)
+void CoreSDKTest::LifecycleReady()
```

Comment:
Yes,if it is not able to connect to the platform/throws any error at the time of execution,it will go to else and print the error status